src/Storages/System/StorageSystemUsers.cpp

```diff
@@ -149,22 +147,24 @@ void StorageSystemUsers::fillData(MutableColumns & res_columns, ContextPtr conte
             {
                 auth_params_json.set("realm", auth_data.getKerberosRealm());
             }
+#if USE_SSL
```

Comment:
You probably also want to wrap enum value `SSL_CERTIFICATE` in src/Access/Common/AuthenticationType.h in `#if USE_SSL`.

Reply:
None of the elements are wrapped in `#if USE_SSL`, so probably not.